crates/holochain/Cargo.toml

```diff
@@ -279,6 +291,23 @@ sqlite = [
 # Extremely verbose wasm memory read/write logging
 wasmer_debug_memory = ["holochain_wasmer_host/debug_memory"]
 
+# Enable wasm compiler
+# Incompatible with "wasmer_wamr"
+wasmer_sys = [
```

Comment:
Can these be enabled together? Is that protected against in wasmer or does Holochain also need to reject a build that enables both?

Reply:
It's checked! Just further down than I'd got to yet. Great :)